src/json/jobs/JobParams.ts

```diff
@@ -0,0 +1,198 @@
+                  "referral.automoderation.complete",
+                  "referral.ended",
+                  "theme.publish.finished"
+                ]
+              }
+            }
+          },
+          additionalProperties: false,
+          required: ["enabledWebhookTypes"]
+        }
+      ],
+      default: true
+    },
+    programs: {
+      oneOf: [
+        {
+          type: "boolean",
+          title: "Program Evaluation Enabled",
+          description:
+            "If true then all applicable active programs will be triggered"
+        },
+        {
+          type: "object",
+          properties: {
+            enabledProgramIds: {
+              type: "array",
+              title: "Enabled Program Ids",
+              items: {
+                type: "string"
+              }
+            }
+          },
+          additionalProperties: false,
+          required: ["enabledProgramIds"]
+        }
+      ],
+      default: true
+    },
+    analytics: {
+      oneOf: [
+        { type: "boolean", title: "Analytics Tracking Enabled" },
+        {
+          type: "object",
+          properties: {
+            enabledAnalyticsEventCollections: {
+              type: "array",
+              title: "Analytics Event Collections",
+              items: {
+                type: "object",
+                properties: {
+                  collectionName: {
+                    type: "string",
+                    title: "Collection Name",
+                    enum: [
+                      "userCreated",
+                      "userReferralProgramLoaded",
+                      "userReferralProgramEngagement",
+                      "userReferralShareLinkClicked",
+                      "userReferralCreated",
+                      "userApprovedReferralCreated",
+                      "userReferralModerated",
+                      "userReferralConverted",
+                      "userApprovedReferralConverted",
+                      "userReferralEnded",
+                      "rewardCreated",
+                      "rewardRedeemed",
+                      "programEvaluated",
+                      "userActivity",
+                      "programGoal"
+                    ]
+                  },
+                  analyticTimestampSource: {
+                    type: "string", // TODO - probably going to delete this and only go with the historic analytic date if we can get it working
+                    enum: ["CURRENT_DATE", "HISTORIC_ANALYTIC_DATE"] // TODO - specs on what HISTORIC_ANALYTIC_DATE means for each event type
+                  }
+                }
+              }
+            }
+          },
+          additionalProperties: false,
+          required: ["enabledAnalyticsEventCollections"]
+        }
+      ]
+    },
+    userMetrics: {
+      oneOf: [{ type: "boolean" }]
+      // TBD? - internal user seen event? - until we include events in the import this isn't relevant
```

Comment:
@jericmason what's the deal here? Formalize it, or remove it.

Reply:
Yeah, that should be removed. It's not relevant to this launch and can always be added later. I'll remove it.